[DM-2970]: Create ui-c8y--services-as-a-plugin.md (#2017)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y--device-management-services-functionality-is-now-available-as-a-plugin.md

---
date: 2024-05-07

> title: Device management services functionality is now available as a plugin.

Reviewer: ```suggestion
title: Device management services functionality is now available as a plugin
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-2970
version:
---

> The functionality for services has been modularized into a plugin. By default, users will have access to services within the device management application. However, if not needed, the functionality can be removed by deactivating the plugin. This approach offers flexibility for users who may or may not require certain functionalities within the application.

Reviewer: ```suggestion
The functionality for services has been modularized into a plugin. By default, users will have access to services within the Device Management application. However, if not needed, the functionality can be removed by deactivating the plugin. This approach offers flexibility for users who may or may not require certain functionalities within the application.
```

> The functionality for services has been modularized into a plugin within the device management application. By default, users will have access to services within the application. However, if needed, the plugin can be removed, thus disabling the services within the application. This approach offers flexibility for users who may or may not require certain functionalities within the application.

Author: ```suggestion
The functionality for services has been modularized into a plugin. By default, users will have access to services within the device management application. However, if not needed, the functionality can be removed by deactivating the plugin. This approach offers flexibility for users who may or may not require certain functionalities within the application.
```
